### PR TITLE
L10n non migration commits

### PR DESCRIPTION
**Pull request by bcolsson**
Removing 2 commits not related to #343 migration

### CODE REVIEW

File: en/mozorg/home-new.ftl [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+home-cta-get-firefox = Get { -brand-name-firefox }
+home-product-pocket-articles = The web’s most intriguing articles
+home-cta-get-pocket = Get { -brand-name-pocket}
+home-product-relay-masks = Easy-to-use email & phone masks
+home-cta-get-relay = Get { -brand-name-relay }
+home-product-vpn-trust = A VPN you can trust
+home-cta-get-vpn = Get { -brand-name-mozilla-vpn }
+
+home-mozilla-takes-bets = “{ -brand-name-mozilla } is taking bets to show the world there’s a business to be made with trustworthy AI. That includes putting things like human rights, data protection and transparency at the core of how these complex systems work.”
+home-politico-cite = Politico
```
Comment (peiying2) [inline en/mozorg/home-new.ftl:28]: Should we add a comment that this is the name of a newspaper/website?

File: en/products/vpn/landing.ftl [hunk 2]
```diff
@@ -146,15 +146,11 @@ vpn-landing-faq-compatibility-question-desc-ios-v3 = <a href="{ $url }">{ -brand
 
 # Variables:
 #   $url (url) - link to https://www.mozilla.org/products/vpn/desktop/linux/
-vpn-landing-faq-compatibility-question-desc-linux-v3 = <a href="{ $url }">{ -brand-name-linux }</a> ({ -brand-name-ubuntu } 18.04 and up)
+vpn-landing-faq-compatibility-question-desc-linux-v4 = <a href="{ $url }">{ -brand-name-linux }</a> ({ -brand-name-ubuntu } 20.04 and up)
```
Comment (peiying2) [inline en/products/vpn/landing.ftl:149]: Have we agreed on not using placeholder for third party brand names?

**Review by peiying2 — COMMENTED** [review]
@craigcook Minor comments. It looks good otherwise.

**Comment by peiying2** [comment]
Oops, looks like this one is the same as PR  #346. I will comment on the other one. We can close this one.

File: en/products/vpn/landing.ftl (hunk 2 above)
Comment (craigcook) [reply en/products/vpn/landing.ftl:149]: I think we should let this remain for now until we can discuss and arrive at a final decision on how to treat third-party brands going forward.

Comment (flodolo) [reply en/products/vpn/landing.ftl:149]:
```suggestion
vpn-landing-faq-compatibility-question-desc-linux-v4 = <a href="{ $url }">Linux</a> (Ubuntu 20.04 and up)
```
https://github.com/mozilla/bedrock/pull/13270

I believe we have a final decision? We should only use terms for Mozilla brand names going forward (new strings).

Comment (alexgibson) [reply en/products/vpn/landing.ftl:149]: If a final decision has been made then we likely need to communicate it more broadly outside of these pull requests. I'll make a follow up PR to remove both of the above brand names ^

Comment (craigcook) [reply en/products/vpn/landing.ftl:149]: I just meant I wanted to talk it over with the team and make sure we all understand, and so we can document the policy somewhere and have a plan going forward. If we're enacting a new policy as of now, we should write that down and tell everyone. 

It's fine for this particular string, but there will certainly be new ones that include third-party brands until we get everyone up to speed. I don't want someone to get yelled for doing it wrong if nobody told them the rules have changed.

**Pull request closed without merging**